erpnext/selling/page/point_of_sale/point_of_sale.py

```diff
@@ -139,8 +139,16 @@ def get_conditions(item_code, serial_no, batch_no, barcode):
 	if serial_no or batch_no or barcode:
 		return "item.name = {0}".format(frappe.db.escape(item_code))
 
-	return """(item.name like {item_code}
-		or item.item_name like {item_code})""".format(item_code = frappe.db.escape('%' + item_code + '%'))
+	cond = "("
+	cond += """item.name like {item_code}
+		or item.item_name like {item_code}""".format(item_code = frappe.db.escape('%' + item_code + '%'))
+	search_fields = frappe.get_all('POS Search Fields', fields = ['fieldname'])
```

Comment:
It would be better if we moved this to a seperate function.

Reply:
Okay will do that